cvat/apps/engine/models.py

```diff
@@ -212,6 +212,7 @@ class Video(models.Model):
     path = models.CharField(max_length=1024, default='')
     width = models.PositiveIntegerField()
     height = models.PositiveIntegerField()
+    fps = models.FloatField(default=25)
```

Comment:
@PMazarovich , need to think one more time, but probably `None` by default is better.